packages/amplify-cli/src/extensions/amplify-helpers/service-select-prompt.js

```diff
@@ -47,10 +47,17 @@ function serviceQuestionWalkthrough(context, supportedServices, category, custom
     });
   }
 
+  let customFlowQuestion;
+  if(context.input.command === 'add'){
+    customFlowQuestion = customQuestion[0];
+  }else{
+    customFlowQuestion = customQuestion[1];
+  }
+
```

Comment:
get rid of this in favor of separate constants as mentioned above

Reply:
fixed